test_data/rules/avoid_function_literals_in_foreach_calls.dart

```diff
@@ -9,12 +9,14 @@ class Person {
 }
 
 void main() {
-  Iterable<String> people;
+  Iterable<String> people = [];
 
-  for (var person in people) { // OK
+  for (var person in people) {
+    // OK
     print('$person!');
   }
-  people.forEach((person) { // LINT
+  people.forEach((person) // LINT
```

Comment:
I'm not sure of the value of moving the opening bracket, but ok.

Reply:
It's a formatting thing.  The formatter moves the comment if it's after the brace, which breaks the expectation.